images/component-server-image/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2006-2020 Talend Inc. - www.talend.com
+#  Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ARG DOCKER_REGISTRY_HOST
+ARG TSBI_IMAGE_PATH
+ARG TSBI_VERSION
+#
+ARG PROJECT_VERSION
+ARG TIMESTAMP
+#
+ARG ADDITIONAL_FILES
+ARG imageWorkDir=/opt/talend/component-kit
+ARG boundPort=8080
+#
+ARG TALEND_JDK_SERIAL_FILTER
+ARG LOGGING_LAYOUT
+
+FROM ${DOCKER_REGISTRY_HOST}${TSBI_IMAGE_PATH}/java8-svc-base:${TSBI_VERSION}
+
+EXPOSE ${boundPort}
+
+COPY ${ADDITIONAL_FILES}/* ${imageWorkDir}/
+
+ENTRYPOINT ["java",
+            "-Djdk.serialFilter=${TALEND_JDK_SERIAL_FILTER:-!*}",
+            "-Djava.security.egd=file:/dev/./urandom",
+            "-Djava.io.tmpdir=${imageWorkDir}/temp",
+            "-Dmeecrowave.home=${imageWorkDir}",
+            "-Dmeecrowave.base=${imageWorkDir}",
+            "-Dlog4j.configurationFile=${imageWorkDir}/conf/log4j2-component-server-${env:LOGGING_LAYOUT}.xml",
+            "-Dgeronimo.metrics.sigar.refreshInterval=0",
+            "-Dtalend.component.exit-on-destroy=true",
+            "-Dtalend.component.manager.services.cache.eviction.defaultEvictionTimeout=30_000",
+            "-Dtalend.component.manager.services.cache.eviction.defaultMaxSize=5_000",
+            "-Dtalend.component.manager.services.cache.eviction.maxDeletionPerEvictionRun=-1",
+            "-cp",
+            "${imageWorkDir}/custom/*:${imageWorkDir}/resources:${imageWorkDir}/classes:${imageWorkDir}/libs/*",
+             "org.apache.meecrowave.runner.Cli",
+             "--http=${boundPort}"
+             ]
+
+LABEL com.talend.name="Talend Component Kit Server"
+LABEL com.talend.application="component-server"
+LABEL com.talend.service="component-server"
+LABEL com.talend.description="Talend Component Kit Backend Server"
+LABEL com.talend.version="${PROJECT_VERSION}"
+LABEL com.talend.build-date="${TIMESTAMP}"
+LABEL com.talend.vendor="Talend"
+LABEL com.talend.maintainer="Talend <support@talend.com>"
+LABEL com.talend.url="https://www.talend.com"
```

Comment:
```suggestion
```
Not necessary, already defined see: https://github.com/Talend/platform-services-microservices#labels

Reply:
removed